docs/_posts/2017-06-29-ahernandez-intro.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  "Community Manager Introduction"
+date:   2017-07-14 16:49:53
+author: Alexander Hernandez
+thumbnail: ahernandez.jpg
+---
+
+Hello Everyone!
+
+My name is Alexander Hernandez and I would like to introduce myself as your OpenGEE Community Manager. I’m currently employed at Thermopylae Sciences and Technology and I’m excited to be a part of this new community. At TST we’re looking forward to improving the project with other Google partners as well as working with the open source community.
+
+As for myself, I worked in the Defense Industry for six years and then transitioned to my current role for the past three. Outside of work you might find me playing guitar in Dallas or out on the golf course on a sunny day. Throughout my career I have been worked with various GIS products so I'm excited to be a part of the community driving OpenGEE.
+
+Normally, an open source group does not get to start with such a strong code foundation. We have inherited a very useful tool from Google and it is my belief that we can find new and interesting uses for it.
+
+If you have been following the project you might have already seen our [Slack Channel](http://slack.opengee.org) but if not then I invite you to join! It has been an active place for discussion and I will do what I can to keep it that way. It is my belief that we should keep the development as open and developer friendly as possible and Slack provides us a good means to do so. If you are interested in using the product and not development the Slack channel is still a great place to discuss your use case or what you can do to get the most of out of your geospatial data.
```

Comment:
Minor: I'd hyphenate 'developer-friendly' since it's one adjective, and add another preposition so it becomes 'interested in using the product and not in development' for clarity.